src/components/application_manager/src/application_impl.cc

```diff
@@ -1112,13 +1155,20 @@ void ApplicationImpl::set_audio_streaming_state(
                  " for non-media application to different from NOT_AUDIBLE");
     return;
   }
-  CurrentHmiState()->set_audio_streaming_state(state);
+
+  // According to proposal SDL-0216 audio and video streaming states should
```

Comment:
The proposal only mentions audio streaming state. It does not mention video.

Reply:
seems that only the comment is wrong. changed in 1b37aa48ddb2672281a97d083bbe0c48453bada8